node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+
+const argv = process.argv;
+const argvs = argv.slice(2);
+const txtFile = argvs[0];
+
+if (txtFile === 'cunningham.txt') {
+  fs.readFile('cunningham.txt', 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+} else if (txtFile === 'hopper.txt') {
+  fs.readFile('hopper.txt', 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+} else if (txtFile === 'dijkstra.txt') {
+  fs.readFile('dijkstra.txt', 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+} else {
+  console.log('invalid entry');
```

Comment:
I would update this so you don't need the if statement, think about how you can use `process.argv` to make that work!